extension/llm/modules/attention.py

```diff
@@ -310,7 +311,9 @@ def false_fn(y):
             self.kv_cache.v_cache.copy_(v)
             self.kv_cache.cache_pos.copy_(cache_pos)
 
-        output = self._sdpa(q, k, v, b, s_x, mask=mask)
+        if input_pos is None:
```

Comment:
This will break the non kv cache version of the text decoder, in that case `input_pos` is also None